src/styles/components/clock.scss

```
@@ -0,0 +1,49 @@
+.clock {
+  padding: 400px;
+
+  &__center {
+    position: relative;
+
+    width: 30px;
+    height: 30px;
+
+    border: 2px solid rgb(115, 115, 255);
+    border-radius: 50%;
+    background-color: #fff;
+    box-shadow: 0 0 0 2px #fff;
+  }
+
+  &__minutes {
+    position: absolute;
+    left: 8px;
+    bottom: 15px;
+    z-index: -1;
+
+    transform-origin: bottom center;
+
+    width: 14px;
+    height: 300px;
+
+    background-color: rgb(115, 115, 255);
+    border-radius: 7px;
+
+    animation: rotate 60s linear infinite;
+  }
+
+  &__seconds {
+    position: absolute;
+    left: 8px;
+    bottom: 15px;
+    z-index: -2;
+
+    transform-origin: bottom center;
+
+    width: 14px;
+    height: 400px;
+
+    background-color: rgb(255, 163, 179);
+    border-radius: 7px;
+
+    animation: rotate 1s linear infinite;
```

same here, too faast
```suggestion
    animation: rotate 60s linear infinite;
```

---

Ah, really, it was a bit stupid. Thank you for review